drivers/nss/lab01/ex1/hello.c

```diff
@@ -0,0 +1,22 @@
+#include <linux/init.h>		// Macros used to mark up functions e.g., __init __exit
+#include <linux/module.h>	// Core header for loading LKMs into the kernel
+#include <linux/kernel.h>	// Contains types, macros, functions for the kernel
+
```

Comment:
Remove comments around includes as they are just making the code harder to read.

Reply:
done.